UnicornEngine/sources/video/Renderer.cpp

```diff
@@ -375,11 +313,11 @@ void Renderer::Render()
             if (!Frame())
                 break;
         }
-        waitAsyncEnd();
+        m_vkLogicalDevice.waitIdle();
     }
 }
 
-void Renderer::onWindowResized(GLFWwindow* window, int width, int height)
+void Renderer::OnWindowResized(GLFWwindow* window, int width, int height)
```

Comment:
Can't be, window initializing much earlier then renderer and if window fucked up, engine will properly shutdown